Documentation/CoreSupport/Tca/Index.rst

```diff
@@ -94,7 +94,7 @@ Here is how such a fied is configured (example from the "tt\_content" table):
 
 .. note::
 
-   This field is traditionally named "l10n\_parent", but any name can be used
+   This field is traditionally named "transOrigPointerField", but any name can be used
```

Comment:
In https://github.com/TYPO3/TYPO3.CMS/blob/master/typo3/sysext/frontend/Configuration/TCA/tt_content.php#L23 it's still `l18n_parent` as in above code example. Of course `l10n_parent` was wrong.

Can you adjust the field name?